R/preprocess.R

```diff
@@ -480,10 +488,9 @@ enw_latest_data <- function(obs) {
   latest_data <- coerce_dt(obs, dates = TRUE)
 
   latest_data <- latest_data[,
-    .SD[report_date == (max(report_date)) | is.na(reference_date)],
+    .SD[report_date == (max(report_date)) & !is.na(reference_date)],
```

Comment:
well this is all very strange! Why not filter out missing reference dates from the get go?